svm/tests/concurrent_tests.rs

```diff
@@ -153,6 +153,8 @@ fn svm_concurrent() {
     let read_account = Pubkey::new_unique();
     let mut account_data = AccountSharedData::default();
     account_data.set_data(AMOUNT.to_le_bytes().to_vec());
+    account_data.set_rent_epoch(u64::MAX);
+    account_data.set_lamports(1);
```

Comment:
the test account is read-only, but this is now necessary because svm would put it in the account cache the first time it loads it in an entry and then assume it had been deallocated the second time it loads it. the svm behavior is correct, because there should be no way to bring an account to zero lamports without it being writable, therefore we should never be able to see one in a read-only context in the first place. our builtins and sysvars have dust lamports